README.md

```diff
@@ -0,0 +1,55 @@
+# terraform-aws-msk
```

Comment:
please have some minimal description what this module do and an maybe dummy code